recipes/glshaderpp/all/test_package/example.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+#include <GL/glew.h>
+#include <GLShaderPP/ShaderProgram.h>
+#include <GLShaderPP/Shader.h>
+#include <GLShaderPP/ShaderException.h>
+
+int main() {
+
+    std::cout << "It compiles so it works!\n";
```

Comment:
is it possible to call some function or instantiate some object, maybe something simple like getting a library version?

Reply:
Next commit instantiates the only possible thing in a headless environment.

Error trace https://github.com/conan-io/conan-center-index/pull/10649#issuecomment-1121284318 (thanks to uilianries) reports a problem with OpenGL's glGetString. This function works on most OS without OpenGL context, but not on macOS.

Creating an OpenGL context in a headless environment needs a lot of dependencies I'm not sure CI image has.